external_contact.go

```diff
@@ -29,3 +29,55 @@ func (c *WorkwxApp) RemarkExternalContact(req *ExternalContactRemark) error {
 	})
 	return err
 }
+
+// GetExternalContactCorpTagList 获取企业标签库
+func (c *WorkwxApp) GetExternalContactCorpTagList(tagID ...string) ([]ExternalContactCorpTagGroup, error) {
+	resp, err := c.execExternalContactGetCorpTagList(reqExternalContactGetCorpTagList{
+		TagID: tagID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return resp.TagGroup, nil
+}
+
+// AddExternalContactCorpTag 添加企业客户标签
+func (c *WorkwxApp) AddExternalContactCorpTag(req ExternalContactCorpTagGroup) ([]ExternalContactCorpTagGroup, error) {
+	resp, err := c.execExternalContactAddCorpTag(reqExternalContactAddCorpTag{
+		ExternalContactCorpTagGroup: req,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return resp.TagGroup, nil
+}
+
+// EditExternalContactCorpTag 编辑企业客户标签
+func (c *WorkwxApp) EditExternalContactCorpTag(id, name string, order uint32) error {
+	_, err := c.execExternalContactEditCorpTag(reqExternalContactEditCorpTag{
+		ID:    id,
+		Name:  name,
+		Order: order,
+	})
+	return err
+}
+
+// DelExternalContactCorpTag 删除企业客户标签
+func (c *WorkwxApp) DelExternalContactCorpTag(tagID, groupID []string) error {
+	_, err := c.execExternalContactDelCorpTag(reqExternalContactDelCorpTag{
+		TagID:   tagID,
+		GroupID: groupID,
+	})
+	return err
+}
+
+// MarkExternalContactTag 标记客户企业标签
+func (c *WorkwxApp) MarkExternalContactTag(userID, externalUserID string, addTag, removeTag []string) error {
```

Comment:
`EditExternalContactTags` might be better, but Tencent already used the "edit" verb for the CRUD endpoints above, so unfortunately we have to live with this "mark" thing. /sigh